pkg/oidc/provider/userinfo.go

```diff
@@ -0,0 +1,125 @@
+package provider
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"slices"
+	"strings"
+
+	"github.com/golang-jwt/jwt"
```

Comment:
Should this be using `github.com/golang-jwt/jwt/v5`?

Reply:
done in https://github.com/rancher/rancher/pull/49558/commits/fbf5ddb3d788905aaa3626ad66f0d9b86cdcac59